server/lib/routes/get-fxa-client-configuration.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (config) {
+  var route = {};
+  route.method = 'get';
+  route.path = '/.well-known/fxa-client-configuration';
+
+  var fxaClientConfig = {
+    /*eslint-disable camelcase */
+    auth_server_endpoint: config.get('fxaccount_url') + '/v1',
```

Comment:
`auth_server_endpoint` or just `auth_server`, or `auth_server_root`? To me endpoint implies a URL to a specific resource, whereas these are partial URLs. The OpenID Connect Discovery Spec use of `_endpoint` implies a full URL to a specific resource too. Examples from the [OpenID Connect Discovery Spec](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse):

```
"authorization_endpoint":
     "https://server.example.com/connect/authorize",
   "token_endpoint":
     "https://server.example.com/connect/token",
   "userinfo_endpoint":
     "https://server.example.com/connect/userinfo",
   "end_session_endpoint":
     "https://server.example.com/connect/end_session",
   "registration_endpoint":
     "https://server.example.com/connect/register",
```